README.md

```diff
@@ -32,6 +32,7 @@ The jsDelivr API allows you to:
  - [list package versions](#list-package-versions)
  - [list package files](#list-package-files)
  - [resolve a version range](#resolve-a-version-range)
+ - [resolve a package entry points](#resolve-a-package-entry-points)
```

Comment:
I think this would be "resolve a package's entry points" or just "resolve package entry points"

Reply:
The second.